src/data/HomePage.yaml

```
@@ -53,51 +53,171 @@ tours:
 
 
 features:
-  - title: High-level Design
-    content: |
-      ## Yes
-      Some discussion about the feature and maybe a code block
-    code:
-
-  - title: Nested Hyperparameters
-    content: |
-      ## Some discussion of the feature
-      Bla Bla
+  - title: Choosing Models
+    content: >-
+      A Model Registry stores detailed metadata for [over 200 models](/machines) and
+      documentation can be searched without loading model code. 
     code: |
-      # 1. Load the Model
-      Tree = @iload DecisionTreeClassifier
-
-      # 2. Create an instance of the model
-      tree = Tree()
-
-      # 3. Wrap the model and data in a machine
-      mach = machine(tree, X, y)
-
-      # 4. Fit the machine
-      fit!(mach)
-
-      # 5. Make predictions
-      yhat = predict(mach, X)
-
-  - title: Neural Networks
-    content: Ding ding.
-    code: 
-
-  - title: Language Models
-    content: Cool
-    code:
+      # match models to data:
+      julia> X, y = @load_iris
+      julia> models(matching(X, y))
+      54-element Vector
+       (name = AdaBoostClassifier, package_name = MLJScikitLearnInterface, ... )
+       (name = AdaBoostStumpClassifier, package_name = DecisionTree, ... )
+       (name = BaggingClassifier, package_name = MLJScikitLearnInterface, ... )
+       ⋮
+      # filter by docstring content:
+      julia> models("pca")
+      (name = PCA, package_name = MultivariateStats, ... )
+      (name = PCADetector, package_name = OutlierDetectionPython, ... )
+
+      # retrieve docs without code import:
+      julia> doc("PCA", pkg="MultivariateStats")
+
+  - title: Meta-algorithms 
+    content: >-
+      For improved composability, and better data hygiene, an extensive number of
+      **meta-algorithms** are implemented as **model wrappers**:
+      <br><br>
+        - [Tuning](https://juliaai.github.io/MLJ.jl/dev/tuning_models/)
+        - Control of [iterative](https://juliaai.github.io/MLJ.jl/dev/controlling_iterative_models/) models
+        - Correction for [class imbalance](https://juliaai.github.io/MLJ.jl/dev/correcting_class_imbalance/)
+        - [Target transforms](https://juliaai.github.io/MLJ.jl/dev/target_transformations/) / inverse transforms
+        - [Thresholding](https://juliaai.github.io/MLJ.jl/dev/thresholding_probabilistic_predictors/) probabilistic predictors
+        - Homogeneous model [ensembling](https://juliaai.github.io/MLJ.jl/dev/homogeneous_ensembles/)
+        - [Recursive feature elimination](https://juliaai.github.io/FeatureSelection.jl/dev/)
```

Is it better to just say
```suggestion
        - [Feature selection](https://juliaai.github.io/FeatureSelection.jl/dev/)
```
Because theoretically, any feature selector dependent on a model (there are many that do I believe) would need to be implemented as a wrapper and these may be added in the future.

---

I only what to list things here that are actual features right now. This is more a list of features tha "design philosophy".

---

Okay. So you have plan to get back to this section as more feature selectors are added.

---

Well, probably. I'm not sure "Features" is ever going to be an exhaustive list. Perhaps we can call it Feature Highlights or Main Features, to clarify this. 

I didn't mention `evaluate`, but I'm assuming this will appear in the Getting Started section anyhow??